### DISCUSSION TITLE
6690 sources

### DISCUSSION THREAD

**Discussion by SaMMy-lacht** [Freetz-NG/freetz-ng#809] [Allgemein]
Also ich habe es nun hinbekommen, dass die 6690 mit der fw 7.50 ohne Fehler durchläuft. Das Gleiche gilt auch für die FW 7.56 natürlich

Hier mal ein kleiner Auszug aus dem log ohne Fehler
[6690_ohne.txt](https://github.com/Freetz-NG/freetz-ng/files/11778744/6690_ohne.txt)

Hier mal ein log mit den Fehlern, die ich schon immer hatte
[6690_fehler.txt](https://github.com/Freetz-NG/freetz-ng/files/11778739/6690_fehler.txt)

Das Ganze ist ein config Fehler, denn wir gemacht haben nicht AVM

Ich will jetzt nur noch testen wie das ganze ausschaut, wenn ich nun die fehlerfreie config auf die 6660 ansetze

EDIT
Super läuft auch durch

**Comment by fda77** [comment]
Das Problem war einfach dass avm noch keine sourcen herausgerückt hatte. Die 6690 hat auch andere als die 6591+6660 , sieht man auch in dfc26ae7ec321783d51cc4bcbacdcce9d858a0d1 6591+6660 sind gleich
Freetz nutzt das was avm als kernelconfig herausrückt, die passt aber so gut wie nie zu den sourcen

**Comment by SaMMy-lacht** [comment]
Die richtige config ist schon bei source-files-FRITZ.Box_6591_Cable-atomp7-07.50.tar.gz dabei ;-) ich meine die für 6690

Auch in der 7.29 würde man sie finden. Grade noch mal geschaut

**Reply by fda77** [reply]
Die GPL sagt nunmal dass man den quellcode bzw patches mit genutzter config herauszurücken hat wenn man sich kostenlos zb am kernel bedient.
Mit viel zu viel Zeit könnte man sich auch selbst eine config zusammenbasteln ... dazu hab ich aber keine lust und hoffe dass avm das mal irgend wann fehlerfrei hinbekommen. Optimal auch noch automatisiert und dann nach noch keine 1gb paket sondern ein 5mb patch...

**Comment by SaMMy-lacht** [comment]
Es ist alles immer dabei ;-). Wir haben nur es übersehen
[6591_6660_6690_config_fehler.txt](https://github.com/Freetz-NG/freetz-ng/files/11778980/6591_6660_6690_config_fehler.txt)

**Comment by fda77** [comment]
Verstehe ich nicht ... wenn du die 3 puma7 src von heute vergleichst, siehst du dass 6591+6660 zu 100% identisch sind
Und ein diff zu 6690 ist größer. 
Die kernelconfig ist aber bei allen 3 gleich. Und mit den alten Sourcen liefen 6591+6660 ohne warnung durch, die 6690 aber viele. Mit den neuen sourcen hat sich das aber erledig

**Comment by SaMMy-lacht** [comment]
deine config-puma7.x86-6690_07.56 105,2 kB (105.244 Bytes) wird zu Fehlern führen
[[genau zu diesen](url)](https://github.com/Freetz-NG/freetz-ng/files/11778739/6690_fehler.txt)

**Reply by fda77** [reply]
Dauert noch bis ich das weiss

**Reply by SaMMy-lacht** [reply]
Ich habe es doch getestet, steht doch schon ganz oben. 
Bei mir war der Test mit 
  usbserial.ko ......................   14.46 Kb 
  ftdi_sio.ko .......................   12.17 Kb 
  pl2303.ko .........................    6.48 Kb

**Reply by fda77** [reply]
Womit? Mit den neuen 6690 sourcen von heute?

**Reply by SaMMy-lacht** [reply]
Die von heute
das gleiche kann ich auch mit der alten machen.

**Comment by fda77** [comment]
Stimmt, wie es aktuell im GIT mit den 7.56 sourcen ist gibt es die ganzen warnugen wegen den sysmbolen, mal wieder Schrott von avm...
Kein Wunder dass das FOS ständig "komische" Dinge macht

> Die richtige config ist schon bei source-files-FRITZ.Box_6591_Cable-atomp7-07.50.tar.gz dabei ;-) ich meine die für 6690
> Auch in der 7.29 würde man sie finden. Grade noch mal geschaut

Und wo findet man die richtige config darin?

**Reply by SaMMy-lacht** [reply]
AVM hat da kein Schrott gemacht. Denn Fehler machen wir ja, sie haben alles richtig gemacht

**Reply by fda77** [reply]
Womit? 

Wenn man die 6690 756 src nimmt und 
```
$ grep AVM_KCONFIG_CONFIG   sources/kernel/linux/.kernelvariables
export AVM_KCONFIG_CONFIG = ../linux/avm/conf/linux-4.9.atomp7
```
findet man die config. Diese config ist ganz offensichtlich falsch (warnungen) -> avm-schrott

Wenn man Langweile hat sucht man in linux/avm/conf/ und findet noch eine "linux-4.9.atomp7.Fritz_Box_HW267x"
Da ich die HWR nicht im kopf hab hab ich nachgeschaut und 6690 ist die hw267. Vielleicht passt die besser? Niemand weiss es. Vielleicht ist das die die du meinst? Weis auch niemand
Laut head ist das eine "Linux/x86 4.9.250 Kernel Configuration" im 6591 paket ... diese kernelversion hat keine der puma 7. Im 6690 src wurde daraus "Linux/x86 4.9.279 Kernel Configuration" und dazu einiges geändert:
```
--- 6591/sources/kernel/linux/avm/conf/linux-4.9.atomp7.Fritz_Box_HW267x        2023-06-14 11:53:58.000000000 +0200
+++ 6690/sources/kernel/linux/avm/conf/linux-4.9.atomp7.Fritz_Box_HW267x        2023-06-07 13:02:33.000000000 +0200
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 4.9.250 Kernel Configuration
+# Linux/x86 4.9.279 Kernel Configuration
 #
 # CONFIG_64BIT is not set
 CONFIG_X86_32=y
@@ -167,7 +167,7 @@
 # CONFIG_NAMESPACES is not set
 CONFIG_SCHED_AUTOGROUP=y
 # CONFIG_SYSFS_DEPRECATED is not set
-# CONFIG_RELAY is not set
+CONFIG_RELAY=y
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_INITRAMFS_SOURCE=""
 CONFIG_RD_GZIP=y
@@ -252,6 +252,8 @@
 CONFIG_HAVE_REGS_AND_STACK_ACCESS_API=y
 CONFIG_HAVE_CLK=y
 CONFIG_HAVE_DMA_API_DEBUG=y
+CONFIG_HAVE_HW_BREAKPOINT=y
+CONFIG_HAVE_MIXED_BREAKPOINTS_REGS=y
 CONFIG_HAVE_USER_RETURN_NOTIFIER=y
 CONFIG_HAVE_PERF_EVENTS_NMI=y
 CONFIG_HAVE_PERF_REGS=y
@@ -431,7 +433,7 @@
 # CONFIG_IOSF_MBI_DEBUG is not set
 # CONFIG_X86_32_IRIS is not set
 CONFIG_SCHED_OMIT_FRAME_POINTER=y
-# CONFIG_HW_BREAKPOINTS is not set
+CONFIG_HW_BREAKPOINTS=y
 # CONFIG_HYPERVISOR_GUEST is not set
 CONFIG_NO_BOOTMEM=y
 # CONFIG_M486 is not set
@@ -540,7 +542,6 @@
 CONFIG_SPARSEMEM_STATIC=y
 CONFIG_HAVE_MEMBLOCK=y
 CONFIG_HAVE_MEMBLOCK_NODE_MAP=y
-CONFIG_ARCH_DISCARD_MEMBLOCK=y
 CONFIG_MEMORY_ISOLATION=y
 # CONFIG_HAVE_BOOTMEM_INFO_NODE is not set
 # CONFIG_MEMORY_HOTPLUG is not set
@@ -1077,6 +1078,11 @@
 # CONFIG_HSR is not set
 # CONFIG_NET_SWITCHDEV is not set
 # CONFIG_NET_L3_MASTER_DEV is not set
+CONFIG_QRTR=y
+# CONFIG_QRTR_TUN is not set
+CONFIG_QRTR_MHI=y
+# CONFIG_QRTR_USB is not set
+# CONFIG_QRTR_FIFO is not set
 # CONFIG_NET_NCSI is not set
 CONFIG_RPS=y
 CONFIG_RFS_ACCEL=y
@@ -1154,16 +1160,33 @@
 CONFIG_WEXT_PROC=y
 CONFIG_WEXT_SPY=y
 CONFIG_WEXT_PRIV=y
-# CONFIG_CFG80211 is not set
+CONFIG_CFG80211=y
+CONFIG_NL80211_TESTMODE=y
+# CONFIG_CFG80211_DEVELOPER_WARNINGS is not set
+# CONFIG_CFG80211_CERTIFICATION_ONUS is not set
+CONFIG_CFG80211_DEFAULT_PS=y
+# CONFIG_CFG80211_DEBUGFS is not set
+# CONFIG_CFG80211_INTERNAL_REGDB is not set
+CONFIG_CFG80211_CRDA_SUPPORT=y
+CONFIG_CFG80211_WEXT=y
+# CONFIG_CFG80211_WEXT_EXPORT is not set
 CONFIG_LIB80211=y
 CONFIG_LIB80211_CRYPT_WEP=y
 CONFIG_LIB80211_CRYPT_CCMP=y
 CONFIG_LIB80211_CRYPT_TKIP=y
 # CONFIG_LIB80211_DEBUG is not set
-
-#
-# CFG80211 needs to be enabled for MAC80211
-#
+CONFIG_MAC80211=y
+CONFIG_MAC80211_HAS_RC=y
+CONFIG_MAC80211_RC_MINSTREL=y
+CONFIG_MAC80211_RC_MINSTREL_HT=y
+# CONFIG_MAC80211_RC_MINSTREL_VHT is not set
+CONFIG_MAC80211_RC_DEFAULT_MINSTREL=y
+CONFIG_MAC80211_RC_DEFAULT="minstrel_ht"
+# CONFIG_MAC80211_MESH is not set
+# CONFIG_MAC80211_LEDS is not set
+# CONFIG_MAC80211_DEBUGFS is not set
+# CONFIG_MAC80211_MESSAGE_TRACING is not set
+# CONFIG_MAC80211_DEBUG_MENU is not set
 CONFIG_MAC80211_STA_HASH_MAX_SIZE=0
 # CONFIG_WIMAX is not set
 # CONFIG_RFKILL is not set
@@ -1212,7 +1235,7 @@
 #
 # Default contiguous memory area size:
 #
-CONFIG_CMA_SIZE_MBYTES=80
+CONFIG_CMA_SIZE_MBYTES=256
 CONFIG_CMA_SIZE_SEL_MBYTES=y
 # CONFIG_CMA_SIZE_SEL_PERCENTAGE is not set
 # CONFIG_CMA_SIZE_SEL_MIN is not set
@@ -1222,6 +1245,28 @@
 #
 # Bus devices
 #
+CONFIG_MHI_BUS=y
+CONFIG_MHI_DEBUG=y
+
+#
+# MHI controllers
+#
+CONFIG_MHI_QTI=y
+
+#
+# MHI device support
+#
+CONFIG_MHI_NETDEV=y
+CONFIG_MHI_UCI=y
+# CONFIG_MHI_SATELLITE is not set
+
+#
+# MHI controllers
+#
+
+#
+# MHI device support
+#
 CONFIG_CONNECTOR=y
 CONFIG_PROC_EVENTS=y
 CONFIG_MTD=y
@@ -1706,27 +1751,89 @@
 # CONFIG_USB_NET_CH9200 is not set
 CONFIG_WLAN=y
 CONFIG_WLAN_VENDOR_ADMTEK=y
+# CONFIG_ADM8211 is not set
 CONFIG_WLAN_VENDOR_ATH=y
 # CONFIG_ATH_DEBUG is not set
+# CONFIG_ATH5K is not set
 # CONFIG_ATH5K_PCI is not set
+# CONFIG_ATH9K is not set
+# CONFIG_ATH9K_HTC is not set
+# CONFIG_CARL9170 is not set
+# CONFIG_ATH6KL is not set
+# CONFIG_AR5523 is not set
+# CONFIG_WIL6210 is not set
+# CONFIG_ATH10K is not set
+# CONFIG_WCN36XX is not set
 CONFIG_WLAN_VENDOR_ATMEL=y
+# CONFIG_ATMEL is not set
+# CONFIG_AT76C50X_USB is not set
 CONFIG_WLAN_VENDOR_BROADCOM=y
+# CONFIG_B43 is not set
+# CONFIG_B43LEGACY is not set
+# CONFIG_BRCMSMAC is not set
+# CONFIG_BRCMFMAC is not set
 CONFIG_WLAN_VENDOR_CISCO=y
+# CONFIG_AIRO is not set
+CONFIG_CNSS_QCN9000=y
+CONFIG_CNSS2=y
+# CONFIG_CNSS2_DEBUG is not set
+# CONFIG_CNSS2_PM is not set
+# CONFIG_CNSS2_PCI_DRIVER is not set
+# CONFIG_CNSS2_CALIBRATION_SUPPORT is not set
+# CONFIG_CNSS2_SMMU is not set
+# CONFIG_CNSS2_RAMDUMP is not set
+CONFIG_CNSS2_GENL=y
 CONFIG_WLAN_VENDOR_INTEL=y
+# CONFIG_IPW2100 is not set
+# CONFIG_IPW2200 is not set
+# CONFIG_IWL4965 is not set
+# CONFIG_IWL3945 is not set
+# CONFIG_IWLWIFI is not set
 CONFIG_WLAN_VENDOR_INTERSIL=y
 CONFIG_HOSTAP=y
 # CONFIG_HOSTAP_FIRMWARE is not set
 # CONFIG_HOSTAP_PLX is not set
 # CONFIG_HOSTAP_PCI is not set
+# CONFIG_HERMES is not set
+# CONFIG_P54_COMMON is not set
 # CONFIG_PRISM54 is not set
 CONFIG_WLAN_VENDOR_MARVELL=y
+# CONFIG_LIBERTAS is not set
+# CONFIG_LIBERTAS_THINFIRM is not set
+# CONFIG_MWIFIEX is not set
+# CONFIG_MWL8K is not set
 CONFIG_WLAN_VENDOR_MEDIATEK=y
+# CONFIG_MT7601U is not set
 CONFIG_WLAN_VENDOR_RALINK=y
+# CONFIG_RT2X00 is not set
 CONFIG_WLAN_VENDOR_REALTEK=y
+# CONFIG_RTL8180 is not set
+# CONFIG_RTL8187 is not set
+CONFIG_RTL_CARDS=y
+# CONFIG_RTL8192CE is not set
+# CONFIG_RTL8192SE is not set
+# CONFIG_RTL8192DE is not set
+# CONFIG_RTL8723AE is not set
+# CONFIG_RTL8723BE is not set
+# CONFIG_RTL8188EE is not set
+# CONFIG_RTL8192EE is not set
+# CONFIG_RTL8821AE is not set
+# CONFIG_RTL8192CU is not set
+# CONFIG_RTL8XXXU is not set
 CONFIG_WLAN_VENDOR_RSI=y
+# CONFIG_RSI_91X is not set
 CONFIG_WLAN_VENDOR_ST=y
+# CONFIG_CW1200 is not set
 CONFIG_WLAN_VENDOR_TI=y
+# CONFIG_WL1251 is not set
+# CONFIG_WL12XX is not set
+# CONFIG_WL18XX is not set
+# CONFIG_WLCORE is not set
 CONFIG_WLAN_VENDOR_ZYDAS=y
+# CONFIG_USB_ZD1201 is not set
+# CONFIG_ZD1211RW is not set
+# CONFIG_MAC80211_HWSIM is not set
+# CONFIG_USB_NET_RNDIS_WLAN is not set

 #
 # Enable WiMAX (Networking options) to see the WiMAX drivers
@@ -1967,6 +2074,25 @@
 CONFIG_AVM_NET_TRACE=y

 #
+# Diag Support
+#
+CONFIG_DIAG_CHAR=m
+
+#
+# DIAG traffic over USB
+#
+
+#
+# DIAG traffic over QRTR
+#
+CONFIG_DIAG_OVER_QRTR=y
+
+#
+# HSIC/SMUX support for DIAG
+#
+CONFIG_DIAGFWD_BRIDGE_CODE=y
+
+#
 # I2C support
 #
 CONFIG_I2C=y
@@ -3179,6 +3305,13 @@
 #
 # Broadcom SoC drivers
 #
+# CONFIG_MSM_SECURE_BUFFER is not set
+# CONFIG_QCOM_DCC is not set
+# CONFIG_MSM_GLINK is not set
+# CONFIG_MSM_GLINK_LOOPBACK_SERVER is not set
+CONFIG_IPQ_SUBSYSTEM_RESTART=y
+CONFIG_IPQ_SUBSYSTEM_RESTART_TEST=m
+# CONFIG_QCOM_TZ_LOG is not set
 # CONFIG_SUNXI_SRAM is not set
 # CONFIG_SOC_TI is not set
 # CONFIG_PM_DEVFREQ is not set
@@ -3203,7 +3336,7 @@
 #
 # PHY Subsystem
 #
-# CONFIG_GENERIC_PHY is not set
+CONFIG_GENERIC_PHY=y
 # CONFIG_PHY_PXA_28NM_HSIC is not set
 # CONFIG_PHY_PXA_28NM_USB2 is not set
 # CONFIG_BCM_KONA_USB2_PHY is not set
@@ -3243,6 +3376,11 @@
 CONFIG_MRPC_AVMCFG_SERVER=y
 # CONFIG_MRPC_AVMCFG_CLIENT is not set
 CONFIG_MRPC_AVM_CRASHLOG=y
+CONFIG_MRPC_AVM_FWINFO_SERVER=y
+
+#
+# Client for AVM firmware info depends on AVM_FW_INFO_EMBED
+#
 CONFIG_AVM_PROM_ENVIRONMENT=y
 CONFIG_AVM_PROM_PLAT=y
 CONFIG_AVM_PROM_GETENV=y
@@ -3579,7 +3717,7 @@
 # CONFIG_BOOTPARAM_SOFTLOCKUP_PANIC is not set
 CONFIG_BOOTPARAM_SOFTLOCKUP_PANIC_VALUE=0
 CONFIG_DETECT_HUNG_TASK=y
-CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=120
+CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=30
 # CONFIG_BOOTPARAM_HUNG_TASK_PANIC is not set
 CONFIG_BOOTPARAM_HUNG_TASK_PANIC_VALUE=0
 # CONFIG_WQ_WATCHDOG is not set
@@ -3664,7 +3802,14 @@
 # CONFIG_TEST_UDELAY is not set
 # CONFIG_MEMTEST is not set
 # CONFIG_TEST_STATIC_KEYS is not set
-# CONFIG_SAMPLES is not set
+CONFIG_SAMPLES=y
+# CONFIG_SAMPLE_KOBJECT is not set
+# CONFIG_SAMPLE_HW_BREAKPOINT is not set
+# CONFIG_SAMPLE_KFIFO is not set
+# CONFIG_SAMPLE_QMI_CLIENT is not set
+# CONFIG_SAMPLE_CONFIGFS is not set
+# CONFIG_SAMPLE_CONNECTOR is not set
+# CONFIG_SAMPLE_SECCOMP is not set
 CONFIG_HAVE_ARCH_KGDB=y
 # CONFIG_KGDB is not set
 CONFIG_ARCH_HAS_UBSAN_SANITIZE_ALL=y
@@ -3981,4 +4126,5 @@
 CONFIG_ARCH_HAS_SG_CHAIN=y
 CONFIG_ARCH_HAS_MMIO_FLUSH=y
 CONFIG_SBITMAP=y
+CONFIG_QMI_ENCDEC=y
 CONFIG_PP_TX_INIT=m
```

In der GPL steht nicht: "packe 1000e configs in ein paket und lass due leute würfeln" ;-)

**Reply by SaMMy-lacht** [reply]
Genau, ;-) ich habe sie auch immer links liegen lassen. Aber war dann doch mal neugierig ;-)

Da ist so vieles drin, wo von wir noch nicht mal was wissen. Wer sucht, findet

**Reply by SaMMy-lacht** [reply]
Es ist aber nicht immer so, dass sie dabei ist oder passt. Bei der 5590 ist so was nicht. Wo habe ich auch mal eine 238 oder so gefunden, da weiß ich nicht mal was das ist

**Reply by fda77** [reply]
Ich verstehe ... du wussteste nicht dass `AVM_KCONFIG_CONFIG = ../linux/avm/conf/linux-4.9.atomp7` die genutzte .config zeigt - und hast dann gesucht welche am besten passt!
Früher hat avm da gern mal über 10 Dateien drin gehabt, aber da waren auch unveröffentlicht Modelle dabei die dann durch die Presse gingen. Daher wird es mittlerweile normal vor dem release gesäubert
Übrigens ist deine "[6591_6660_6690_config_fehler.txt](https://github.com/Freetz-NG/freetz-ng/files/11778980/6591_6660_6690_config_fehler.txt)" ein bisschen anders als fafd3cddf8933c3218fe2b6f1ee85bf78d998b64 , ich vermute weil du die aus dem alten paket genommen hast

**Reply by fda77** [reply]
5590 hat 3 cpus  - mit 3 betriebsystemen. Manchmal kommt hinter die HWR ein a/b oder ne ganz eigene
Die hat nunmal mehrer HWR. Das hast du vermutlich von Erny, das gehört nämlich zu den Dingen die er nicht kapiert ;-)

**Reply by SaMMy-lacht** [reply]
Nein, ich habe immer die erst genommen. Aber dann wollte ich wann wissen, was das für ein Gerät ist, die da mit HW bei liegt. Und habe geschaut. Und heute bei der 6690 noch mal geschaut und da passte es. Testbau gemacht und sehe da keine Fehler mehr .

Nein ich starte immer make kernel-menuconfig baue das ein was ich brauche (autoscript)
und das habe ich mit beiden Datein gemach

**Reply by fda77** [reply]
Wenn du gerne und viel compilierst, bau doch mal wireguard (ab kernel v3) überall ein, das könnte noch bei anderen fehlen: 093e9b0b678416cc53970539a276dc10cf8699a1

**Reply by SaMMy-lacht** [reply]
Ich kann dir nur bezüglich der Config für die 6591 und 6660 noch sagen, dass die auch läuft von der 6690 

denn ich habe die 6591 und 6660 auch auf die 6690 greifen lassen

**Reply by fda77** [reply]
Ich hatte wie oben geschrieben ein diff zwischen 6591/6660 und 6690 gemacht und da waren schon einige Unterschiede. Falls es mal replace kernel für v4 kernel geben wird ist es besser wenn es nicht die gleichen src sind

**Reply by SaMMy-lacht** [reply]
Glaubst du das es das mal wieder geben wird?

**Comment by SaMMy-lacht** [comment]
linux-4.4.scrpn.Fritz_Box_HW238
ist eine 7490
https://boxmatrix.info/wiki/FRITZ!Box_7490

**Reply by fda77** [reply]
Und? bzw Aber?
Scropion ist die 2. cpu der 7490 die Wlan macht. Das 2. BS hat eine eigen HWR. Und Freetz ändert daran nichts. Wie bei allen Puma nur Atom geändert wird und Arm nicht.
Für die 7490 gibt es einen riesigen, ein paar Jahre alten PR/Fork irgendwo auf Github von Openwrt ... ist wohl nicht so einfach

**Reply by fda77** [reply]
https://github.com/hvegh/openwrt/tree/for_fritzbox7490  scorpion ist das wasp image

**Reply by SaMMy-lacht** [reply]
Diese Datei liegt bei allen neuen sources dabei 
linux-4.4.scrpn.Fritz_Box_HW238
daher wudert mich das es eine 7490 ist

**Reply by fda77** [reply]
Avm halt ^^
Daher ist die "Wlan geht bei der 7490" ein hinweis darauf wass die Wlan cpu abgeraucht ist oder nicht genug strom bekommt
Dann könnte man zb auch einfach openwrt installieren, ohne sich die arbeit mit dem wasp image zu machen :)

**Reply by SaMMy-lacht** [reply]
Ich lasse mich überraschen!

**Reply by fda77** [reply]
Womit? Wenn du auf deinen boxmatrix link klickst und dann "ProductID" siehst du dass 3490 5490 5491 7490 diese HWR für die wlan-cpu verwenden. Für Freetz egal

**Reply by fda77** [reply]
Bzw auf der "HWR-Names" Seite

> Dummy for the 2nd Linux on the Scorpion WiSoC of the 3490, 5490, 5491, 7490

**Comment by SaMMy-lacht** [comment]
Mal eine Frage, willst du denn Fehler in der FW 7.2X drin lassen? Bezüglich der 6690.

**Reply by fda77** [reply]
hast du wireguard für alles compilieren können, ohne sowas wie a7e1ce1daf4c5fbe45e5e47cede513b5f0e72e4c ?

**Reply by SaMMy-lacht** [reply]
Schau liste ;-)
https://github.com/Freetz-NG/freetz-ng/commit/a7e1ce1daf4c5fbe45e5e47cede513b5f0e72e4c#commitcomment-118723194
